src/lib/modules/ens/embarkjs.js

```diff
@@ -153,13 +153,25 @@ __embarkENS.setProvider = function (config) {
   self.registration = config.registration;
   self.env = config.env;
   EmbarkJS.onReady(() => {
-    web3.eth.net.getId()
+    EmbarkJS.Blockchain.blockchainConnector.getNetworkId()
       .then((id) => {
         const registryAddress = self.registryAddresses[id] || config.registryAddress;
         self._isAvailable = true;
-        self.ens = new EmbarkJS.Blockchain.Contract({abi: config.registryAbi, address: registryAddress, web3: web3});
-        self.registrar = new EmbarkJS.Blockchain.Contract({abi: config.registrarAbi, address: config.registrarAddress, web3: web3});
-        self.resolver = new EmbarkJS.Blockchain.Contract({abi: config.resolverAbi, address: config.resolverAddress, web3: web3});
+        self.ens = new EmbarkJS.Blockchain.Contract({
+          abi: config.registryAbi,
+          address: registryAddress,
+          web3: EmbarkJS.Blockchain.blockchainConnector.getInstance()
+        });
+        self.registrar = new EmbarkJS.Blockchain.Contract({
+          abi: config.registrarAbi,
+          address: config.registrarAddress,
+          web3: EmbarkJS.Blockchain.blockchainConnector.getInstance()
+        });
+        self.resolver = new EmbarkJS.Blockchain.Contract({
+          abi: config.resolverAbi,
+          address: config.resolverAddress,
+          web3: EmbarkJS.Blockchain.blockchainConnector.getInstance()
+        });
```

Comment:
Here is just me putting the params on multiple lines and replacing web by the blockchainConnector's function